src/routers/MultiRouteRouter.js

```diff
@@ -35,16 +36,21 @@ class MultiRouteRouter extends ApplicationRouter<Array<Object>> {
     }
 
     // multiple destinations given
```

Comment:
Comment should address that the variable is an array of routes to each destinations